addon/popup.css

```diff
@@ -104,6 +104,48 @@ body {
   margin-inline-end: 8px;
 }
 
+.tab__dismiss {
+  position: relative;
+  background: transparent;
+  display: none;
+  margin-left: auto;
+  border-radius: 10%;
+  height: 20px;
+  width: 20px;
+}
+
+.tab__dismiss:hover,
+.tab__dismiss:focus {
+  background: rgba(224, 224, 225, 0.9);
+}
+
+#panel.dark-theme .tab__dismiss:hover,
+#panel.dark-theme .tab__dismiss:focus {
+  background: --dark-theme-highlight-color;
+}
+
+.tab:hover .tab__dismiss,
+.tab:focus .tab__dismiss {
+  display: inline-flex;
+}
+
+.tab__dismiss::after {
+  position: absolute;
+  font-size: 18px;
+  color: rgb(83, 85, 85);
+  content: '\d7';  /* cross symbol */
+  top: -1px;
+  bottom: 0;
+  left: -1px;
+  right: 0;
+  line-height: 20px;
+  text-align: center;
+}
+
+#panel.dark-theme .tab__dismiss::after {
```

Comment:
I know there is a bit of dark theme stuff here, but that is fine. Make sure it all integrates smoothly when fixing the conflicts in the dark theme pr.